CONTRIBUTING.md

```diff
@@ -28,5 +28,25 @@ We use GitHub issues to track public bugs. Please ensure your description is cle
 * "Attractive"
 * Do not use the optional parameters of `setTimeout` and `setInterval`
 
+## Testing
+
+To run tests in the browser, run
+
+```
+npm run test:server
+```
+
+Open http://localhost:8081/test. Anytime you change test files, the tests will automatically be rerun, no refresh
+required! You can even click a specific section to only run those tests.
+
+To run tests in the CLI, run
+
+```
+npm run test
+```
+
+This will fail on some errors that npm run test:server won't, so it's recommended that you run this command after you've
+finished developing on tests with npm run test:server, if you chose that method.
```

Comment:
I'm 90% sure this is because of strict mode.
If you add 'use strict' to this top of that file and re-run; you should see identical errors.  
If thats the case, please file a ticket to enable strict mode in our tests